cmd/globals.go

```diff
@@ -167,6 +167,16 @@ var (
 	colorYellow = color.New(color.FgYellow).SprintfFunc()
 )
 
+// Secure Go implementations of TLS ciphers
+var tlsCipherSuites = []uint16{
```

Comment:
If this is more secure TLS configuration, how about making this default cipher suite for http server in case of TLS enabled?  You always has an option to override this default behavior.

Reply:
Don't understand - what do you mean with "making this the default cipher suites".  These ciphers are used as the "default" cipher suites for the server and the gateway. Or do I miss something?
See https://github.com/minio/minio/pull/5245/files#diff-736572b438aa05056f334265beb9507bR158

Reply:
My suggestion is below
1. move `tlsCipherSuites` to `pkg/http/server.go` itself i.e. no definition in `cmd/*.go`
2. there is no `cipherSuites` argument to `NewServer()` i.e. `tlsCipherSuites` is default.
3. if user wants to override this, he could directly set `Server.TLSConfig` accordingly to his need.
4. this way there is no change in `cmd/*.go`

Reply:
That's exactly how the patch looked before but harsha mentioned:
>We should take this list as perhaps an argument to NewServer() function, in future we might need to change these as well. We can take this as an input from top level and not default/restrict this package to the following ciphers.

Reply:
@harshavardhana As the approach provides support to the use case you mentioned, do you have any other concern to do the change?

Reply:
That won't be a problem, since the TLSConfig can be overriden. I guess having a default is fine for now.